config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/builder/AnnotableEntity.java

```diff
@@ -50,6 +58,7 @@
             AnnotatedEntity<GatewayEntity> annotatedEntity = new AnnotatedEntity(this);
             annotations.forEach(annotation -> {
                 if (ANNOTATION_TYPE_BUNDLE.equalsIgnoreCase(annotation.getType())) {
+                    annotatedEntity.setMetadataId(annotation.getId());
```

Comment:
its bundle id

Reply:
@uppoju The attribute name is `id` both while specifying in the annotation and in the metadata.
If `id` is added to `@bundle` annotation, it will be Metadata ID. If added to `@bundle-entity` it will be BundleEntity ID.

Reply:
i mean, the variable name.
Instead of metadataId it should be bundleId

Reply:
Metadata Id is more self explanatory.